newIDE/app/src/Profile/AuthenticatedUserContext.js

```diff
@@ -17,6 +18,8 @@ export type AuthenticatedUser = {|
   loginState: null | 'loggingIn' | 'done',
   badges: ?Array<Badge>,
   cloudProjects: ?Array<CloudProjectWithUserAccessInfo>,
+  ownedAssetPackIds: ?Array<string>,
+  ownedAssetShortHeaders: ?Array<AssetShortHeader>,
```

Comment:
Should this be:
```suggestion
  ownedPrivateAssetPackIds: ?Array<string>,
  ownedPrivateAssetShortHeaders: ?Array<AssetShortHeader>,
```

Reply:
or 
```suggestion
  receivedPrivateAssetPackIds: ?Array<string>,
  receivedPrivateAssetShortHeaders: ?Array<AssetShortHeader>,
```

Reply:
yup, I'm renaming everything to be clearer